src/components/CohortBuilder/Summary/DemographicChart.js

```diff
@@ -46,43 +47,50 @@ const DemographicChart = ({ data, theme }) => (
 );
 
 export const demographicQuery = sqon => ({
-  query: `query ($sqon: JSON) {
-    participant {
-      hits(filters: $sqon) {
-        edges {
-          node {
-            race
-            ethnicity
-            gender
-            family {
-              family_compositions {
-                hits {
-                  edges {
-                    node {
-                      composition
-                    }
-                  }
-                }
-              }
+  query: gql`
```

Comment:
I have no idea why we never used this, just one of those things that should have been there since the beginning, but not somehow...